enhancements/sig-architecture/146-fleet-namespaces/README.md

```diff
@@ -0,0 +1,182 @@
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Introduces a global mechanism for defining and managing namespaces on Managed Clusters directly from the hub. Namespaces can now be associated with a ManagedClusterSet, enabling centralized application of Kubernetes access controls through ClusterPermissions. Both the namespaces and their access controls are bound and managed by the ManagedClusterSet, and applied equally to all Managed Clusters in the ManagedClusterSet, granting multi-cluster governance and improving security consistency across fleets.
+
+## Motivation
+ 
+A new global mechanism now enables the centralized definition and management of namespaces and their associated permissions across all clusters within a ManagedClusterSet. Previously, ManifestWork and ClusterPermission resources were required separately and operated independently. With this enhancement, namespaces and access controls are automatically and consistently applied to every managed cluster in the set, simplifying security management at scale, ensuring namespace consistency, and supporting seamless migration and improved visibility for workloads such as Virtual Machines.
+
+This is not meant to manage all namespaces on the Managed Cluster, just to create uniform namespace management across Managed Clusters in a Managed Cluster Set
+
+### Goals
+
+1. Create a new CRD API for a namespace that spans all clusters in a Managed Cluster Set (Global Namespace?!?)
+2. Associate Cluster Permissions to the Managed Cluster Set in terms of the Global Namespace
+3. Work with the existing RBAC tooling
+
+### Non-Goals
+
+1. Managing all namespaces on the Managed Cluster
+
+## Proposal
+
+This is where we get down to the nitty gritty of what the proposal actually is.
+
+### User Stories
+
+#### Story 1
+Create a Global Namespace, this namespace will be created on all Managed Clusters in a Managed Cluster Set, and 
+the appropriate Cluster Permissions will be applied.
```

Comment:
if we create a managedclusterbinding that binds a clusterset to a namespace, does it mean we should sync the namespace that creates managedclusterbinding to all clusters? It seems we can add a spec field in managedclusterbinding to achieve that.